src/components/popover/popover/popover.ts

```diff
@@ -379,55 +359,83 @@ export class SbbTooltipElement extends LitElement {
   // To avoid entering a corrupt state, exit when state is not expected.
-  private _onTooltipAnimationEnd(event: AnimationEvent): void {
+  private _onPopoverAnimationEnd(event: AnimationEvent): void {
     if (event.animationName === 'open' && this._state === 'opening') {
       this._state = 'opened';
       this._didOpen.emit();
       this.inert = false;
-      this._setTooltipFocus();
-      this._focusHandler.trap(this);
+      this._setPopoverFocus();
+      this._focusHandler.trap(this, {
+        postFilter: (el) => el !== this._overlay,
+      });
       this._attachWindowEvents();
     } else if (event.animationName === 'close' && this._state === 'closing') {
       this._state = 'closed';
       this._overlay?.firstElementChild?.scrollTo(0, 0);
+      this._overlay?.removeAttribute('tabindex');
+
+      if (!this._skipCloseFocus) {
+        const elementToFocus = this._nextFocusedElement || this._triggerElement;
 
-      const elementToFocus = this._nextFocusedElement || this._triggerElement;
+        setModalityOnNextFocus(elementToFocus);
+        // To enable focusing other element than the trigger, we need to call focus() a second time.
+        elementToFocus?.focus();
+      }
 
-      setModalityOnNextFocus(elementToFocus);
-      // To enable focusing other element than the trigger, we need to call focus() a second time.
-      elementToFocus?.focus();
-      this._didClose.emit({ closeTarget: this._tooltipCloseElement });
+      this._didClose.emit({ closeTarget: this._popoverCloseElement });
       this._windowEventsController?.abort();
       this._focusHandler.disconnect();
     }
   }
 
   // Set focus on the first focusable element.
-  private _setTooltipFocus(): void {
+  private _setPopoverFocus(): void {
     const firstFocusable =
-      this.shadowRoot!.querySelector<HTMLElement>('[sbb-tooltip-close]') ||
+      this.shadowRoot!.querySelector<HTMLElement>('[sbb-popover-close]') ||
       getFirstFocusableElement(
         Array.from(this.children).filter((e): e is HTMLElement => e instanceof window.HTMLElement),
       );
     if (firstFocusable) {
       setModalityOnNextFocus(firstFocusable);
       firstFocusable.focus();
+    } else {
+      this._overlay.setAttribute('tabindex', '0');
+      setModalityOnNextFocus(this._overlay);
+      this._overlay.focus();
+      this._overlay.removeEventListener('blur', this._blurListener);
+      this._overlay.addEventListener('blur', this._blurListener, {
+        signal: this._popoverController.signal,
+      });
     }
   }
 
-  private _setTooltipPosition(): void {
+  private _blurListener = (): void => {
+    // When a blur occurs, we know that the popover has to be closed, because there are no interactive elements inside the popover.
+    // We have to ensure that window / tab change doesn't trigger closing. This can be achieved by checking visibilityState, which only works with setTimeout().
```

Comment:
Nitpick; This line is longer than 100 characters and wraps around in the GitHub code comparison view. 😅

Reply:
fixed :)